lms/djangoapps/certificates/utils.py

```diff
@@ -69,6 +69,19 @@ def has_html_certificates_enabled(course):
     return course.cert_html_view_enabled
 
 
+def has_html_certificates_enabled_from_course_overview(course_overview):
```

Comment:
One of two slightly duplicated functions introduced in this PR. 

This uses a course_overview over the course object (from the modulestore) to determine if HTML certificates are enabled for a course. I think the original `has_html_certificates_enabled` will eventually go away as the refactor continues. At which point I can likely rename this function to the original name. I will be making these changes in smaller chunks for reviewing ease and testing purposes.